src/Microsoft.Azure.SignalR.Management/ServiceManagerBuilder.cs

```diff
@@ -11,6 +13,7 @@ namespace Microsoft.Azure.SignalR.Management
     public class ServiceManagerBuilder : IServiceManagerBuilder
     {
         private readonly ServiceManagerOptions _options = new ServiceManagerOptions();
+        private Assembly _assembly = Assembly.GetExecutingAssembly();
```

Comment:
>GetExecutingAssembly [](start = 46, length = 20)

no need?

Reply:
We need it. If `WithCallingAssembly`() isn't called, we need to track the Management SDK itself.
Will move it to `Build()` to avoid calling twice.